package.json

```diff
@@ -2,15 +2,6 @@
   "name": "dc-addons-bubble-chart",
   "version": "0.1.3",
   "dependencies": {
-    "crossfilter": "^1.3.11",
-    "d3": "^5.5.0",
-    "dc": "^3.0.2",
```

Comment:
This package does not actually import dc or d3 so it doesn't need to have them as dependencies